src/scopes.ts

```diff
@@ -478,6 +481,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     description: 'View the consent manager configuration.',
     products: [TranscendProduct.ConsentManager],
   },
+  [ScopeName.ManageDocuments]: {
+    dependencies: [],
+    description: 'Edit the configurations on documents and document templates.',
+    title: 'Manage Documents',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.Assessments],
```

Comment:
I think it should be either Admin or Assessments. Will wait for Mike's feedback here

Reply:
i think this is right to make the new product, but lets also tag it with `TranscendProduct.DataMapping` as this will be included in that offering for existing customers for the time being

Reply:
Sounds good.

Reply:
done